### PR TITLE
GB-1918

### PR DESCRIPTION
**Pull request by osman**
add support for bring to top

### CODE REVIEW

**Comment by osman** [comment]
@effervescentia

File: src/main/java/com/groupbyinc/api/Query.java [hunk 1]
```diff
@@ -1208,4 +1225,22 @@ public Query setBiasing(Biasing biasing) {
...
+     *
+     * </code>
+     *
+     * @param bringToTop
+     *         Any number of product IDs to bring to the top of the result set.
+     *
+     * @return
+     */
+    public Query setBringToTop(String... bringToTop) {
+        CollectionUtils.addAll(this.biasing.getBringToTop(), bringToTop);
```
Comment (effervescentia) [inline src/main/java/com/groupbyinc/api/Query.java:1243]: I think having a function called _set_ that does an addAll is confusing

**Comment by effervescentia** [comment]
@osman looks good other than my one comment

**Comment by osman** [comment]
@effervescentia needs api changes in api-php and api-dotnet

**Pull request merged**